rados/rados_getaddrs_test.go

```diff
@@ -0,0 +1,16 @@
+//go:build ceph_preview
+// +build ceph_preview
+
+package rados
+
+import (
+	"github.com/stretchr/testify/assert"
+)
+
+func (suite *RadosTestSuite) TestGetAddrs() {
+	suite.SetupConnection()
+
+	addrs, err := suite.conn.GetAddrs()
+	assert.NoError(suite.T(), err)
+	assert.NotEmpty(suite.T(), addrs, "rados_getaddrs")
```

Comment:
I suppose there's nothing else we can assert about the content of this string, right?

Reply:
Most likely not, unless we parse the string. But if so, then that should be done by `GetAddrs` itself?

Reply:
Agreed, as noted in the other comment let's assume the string is opaque.

And... that's fine. It was somewhat of a rhetorical question... but I felt the need to write it anyway. :-)